Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -1,23 +1,28 @@
+import KsApi
+import Library
 import Prelude
 import ReactiveSwift
 import Result
-import KsApi
-import Library
 
 public protocol SettingsAccountViewModelInputs {
-
-  func settingsCellTapped(cellType: SettingsAccountCellType)
+  func didConfirmChangeCurrency()
+  func dismissPickerTap()
+  func didSelectRow(cellType: SettingsAccountCellType)
+  func showChangeCurrencyAlert(for currency: Currency)
   func viewDidLoad()
 }
 
 public protocol SettingsAccountViewModelOutputs {
-
-  var reloadData: Signal<Void, NoError> { get }
+  var dismissCurrencyPicker: Signal<Void, NoError> { get }
+  var presentCurrencyPicker: Signal<Bool, NoError> { get }
+  var reloadData: Signal<(User, Currency), NoError> { get }
+  var showAlert: Signal<(), NoError> { get }
   var transitionToViewController: Signal<UIViewController, NoError> { get }
+  var updateCurrency: Signal<Currency, NoError> { get }
```

Comment:
I think we can remove this output since we're using `reloadData` to signal when the currency has been updated.